sdk/core/core-http/lib/util/utils.ts

```diff
@@ -140,7 +153,7 @@ export function executePromisesSequentially(promiseFactories: Array<any>, kickst
  *
  * @returns {object} Returns the merged target object.
  */
-export function mergeObjects(source: { [key: string]: any; }, target: { [key: string]: any; }) {
+export function mergeObjects(source: { [key: string]: any }, target: { [key: string]: any }) {
```

Comment:
I think this helper is just the standard Object.assign. Remove this and use that instead.